hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -814,6 +814,20 @@
       value.
     </description>
   </property>
+  <property>
```

Comment:
I would prefer to use the new, Java based configuration API:

https://cwiki.apache.org/confluence/display/HADOOP/Java-based+configuration+API